microcosm_daemon/health_reporter.py

```diff
@@ -35,7 +35,10 @@ def __call__(self, health, prev_health, errors):
             if isinstance(error, ExitError):
                 continue
 
-            logger.warn(f"Caught error during state evaluation: {error}", exc_info=True)
+            logger.exception(
```

Comment:
Those logging statements triggered errors from our [flake8-logging-format](https://github.com/globality-corp/flake8-logging-format). Not sure why it didn't trigger on the original PR.